### ISSUE TITLE
spike to investigate (feasibility, efforts) multiple source case for content portal

### ISSUE THREAD

**Issue by chianger** [mozilla-tw/FirefoxLite#3192]
#2712 is WIP while it's targeting on single source use case. 
Soon we will have more content partners so that multiple source (either leveraging a backend or just client side) must be taken care of. 
Expected outcome from this spike:
1. Proposal of the architecture
2. Efforts estimation for the implementation and maintenance.

**Comment by chianger** [comment]
downgrade to P2 given that we can do AB testing for different source before deciding to do aggregation.

**Comment by chianger** [comment]
Upgrade to P1. 
We no longer consider aggregation of multiple source. Instead, we want to provide multiple category of news content to users.

**Comment by chianger** [comment]
https://drive.google.com/drive/folders/1FCrvI9S9UUhc_aSs9zmn5at4dm8fne8i  spec in this folder

**Comment by cnevinc** [comment]
Some findings:
https://docs.google.com/document/d/1TAagWc86y3KmY4Df5RLqtgTI99bA4GmIEaBXyztLmR0/edit?usp=sharing

TLDR;
DB: not able to categorize/filter/customize news
Newspoint: able to filter by city, state, category, language. Some fields need verification.

Time estimation
UI discussion: 6 hours
API discussion: 6 hours. time depend on partner's schedule
UI Implemtation: 1-3 sprint depending on the design. size XL.

**Comment by Jacklin-ux** [comment]
**Note**
1.Newspoint
(Category with English)
"movie-reviews","politics","career","education","entertainment","regional","videos","astrology","india","photos","automobile","world","crime","events","top-news","sports","business","health","technology","City","food","lifestyle","cricket","science","travel","jokes"
(Language)
english, hindi, marathi, kannada, gujarati, malayalam, tamil, telugu, urdu, nepali, asamiya, punjabi, gujarati, bengali

2.DB
Asking partner...

**Comment by chianger** [comment]
one source is good enough for now.
